float32 deltaX
float32 deltaY
float32 deltaTheta
float32 hertz
uint32  isSafe
uint32  wristBumperState
